backend/src/main/resources/application-okta-stg.yaml

```diff
@@ -3,6 +3,7 @@
 okta:
   oauth2:
     client-id: 0oa62qncijWSeQMuc4h6
-simple-report.authorization:
+simple-report:
```

Comment:
I was scratching my head why you would have done this, and then I realized the edit sequence involved. 😁